packages/foundation/src/createComponent.tsx

```diff
@@ -36,21 +37,23 @@ export function createComponent<
   TViewProps extends TComponentProps = TComponentProps,
   TStatics = {}
 >(
-  component: IComponentOptions<TComponentProps, TTokens, TStyleSet, TViewProps, TStatics>
+  view: IViewComponent<TViewProps>,
+  options: IComponentOptions<TComponentProps, TTokens, TStyleSet, TViewProps, TStatics> = {}
 ): React.FunctionComponent<TComponentProps> & TStatics {
-  const { factoryOptions = {} } = component;
+  const { factoryOptions = {} } = options;
   const { defaultProp } = factoryOptions;
+  const displayName = (options && options.displayName) || view.name;
```

Comment:
Something about this feels weird to me. The fact that the customization can come from either displayName or view.name confuses me a bit, especially because I don't actually see view.name as an explicit property of IViewComponent.

Reply:
`name` is a JavaScript prototype. This naming change is more symmetric with `styled`'s behavior.

Reply:
That's good to know about the name!

Reply:
It's a good point about using it for customizations though.. we shouldn't use `name` there because it will be affected by minification and cause confusion. I'll make a quick change to only use `options.displayName` for Customizations.